CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
 option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
 option(ENABLE_THREAD_COUNT "Turn on correct threads shutdown checks" OFF)
 option(FAST_BUILD "Turn on fast build" ON)
+option(USE_CCACHE "Turn on ccache usage" ON)
+option(USE_DISTCC "Turn on distributed build_usage" ON)
```

Comment:
Note that enabling this flag switch automatic distributed build id developer use multijobs make.
In case if developer use one job make - no distribution will be enabled.